CCDB/test/testCcdbApiMultipleUrls.cxx

```diff
@@ -0,0 +1,126 @@
+// Copyright 2019-2021 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   testCcdbApiMultipleUrls.cxx
+/// \author Rafał Mucha
+///
+
+#define BOOST_TEST_MODULE CCDB
+#define BOOST_TEST_MAIN
+#define BOOST_TEST_DYN_LINK
+
+#include "CCDB/CcdbApi.h"
```

Comment:
That's a lot of includes. Are they all necessary ?

Reply:
No, you are right. I have left necessary